tests/integration/targets/server/tasks/test_primary_ips.yml

```diff
@@ -57,15 +57,13 @@
     that:
       - result is changed
 
-- name: cleanup server with primary ips
-  hetzner.hcloud.server:
-    name: "{{ hcloud_server_name }}"
-    state: absent
-
 - name: cleanup test create primary ips
   hetzner.hcloud.primary_ip:
     name: "{{ hcloud_server_name }}v4"
     state: absent
+  until: result is not failed
+  retries: 5
+  delay: 2
```

Comment:
Why is this required?

Reply:
Not sure all the other tests had it except that one.  I can remove but was just standardizing the removals.
https://github.com/ansible-collections/hetzner.hcloud/blob/694a238a386f6b03013f8e852e22c48f0d40f58d/tests/integration/targets/server/tasks/test_primary_ips.yml#L65C1-L82C11